sys/const.go

```diff
@@ -264,3 +265,12 @@ var (
 		"wavelet.verify.ed25519":  5000, // TODO: Review
 	}
 )
+
+// String converts a given tag to a string.
+func (tag Tag) String() string {
+	if tag < 0 || tag > 4 { // Check out of bounds
+		return "" // Return invalid tag
+	}
+
+	return []string{"TagNop", "TagTransfer", "TagContract", "TagStake", "TagBatch"}[tag] // Return tag
```

Comment:
We should rename the string tag literals here to "nop", "transfer", "contract", "stake", "batch".